src/vs/base/browser/dom.ts

```diff
@@ -1040,6 +1040,14 @@ export function join(nodes: Node[], separator: Node | string): Node[] {
 	return result;
 }
 
+export function toggleVisibility(shouldShow: boolean, ...elements: HTMLElement[]): void {
+	if (shouldShow) {
+		show(...elements);
+	} else {
+		hide(...elements);
+	}
+}
```

Comment:
This looks like setting visibility, not toggling. I was expecting: `toggleVisibility(elements: HTMLElement[], shouldShow?: boolean)`, like how `classList.toggle` works